docs/user/gcp/customization.md

```diff
@@ -4,6 +4,7 @@ Beyond the [platform-agnostic `install-config.yaml` properties](../customization
 
 * `projectID` (required string): The project where the cluster should be created.
 * `region` (required string): The GCP region where the cluster should be created.
+* `publiczoneid` (optional string): The ID of the public DNS zone name (this is the GCP managed name).
```

Comment:
the field mentioned here is incorrect, the json/yaml field name is actually `publicZoneID`

Reply:
Fixed.